astrokat/test/test_offline_observe.py

```diff
@@ -159,4 +159,4 @@ def test_image_sim(self):
         # do no need to super acurate with this target to allow
         # for slew time discrepancies
         self.assertIn("T4R02C02 observed", result)
-        self.assertIn("T4R02C04 observed for 180.0 sec", result)
+        self.assertIn("T4R02C04 observed for 360.0 sec", result)
```

Comment:
Will we need to update the MKAT AQF tests?

Reply:
@ajoubertza I doubt i had a look at the AQF tests it seems like slew time is not being tested there.